projects/demo/src/modules/components/button/examples/5/index.html

```diff
@@ -0,0 +1 @@
+<button tuiButton>Default options</button>
```

Comment:
Isn't it exactly NOT the default options? 🙂

Reply:
u r right) renamed